[🐛] JuiceShop Docker Image Crashing on OpenShift since 9.1.0

<!--🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅

To expedite issue processing please search open and closed issues before submitting a new one.
Existing issues often contain information about workarounds, resolution, or progress updates.

🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅-->

# :bug: Bug report

## Description

When running JuiceShop on OpenShift the Container crashes directly.

### Is this a regression?

Yes, the previous version in which this bug was not present was: `9.0.1`

## :microscope: Minimal Reproduction

OpenShift with default security context setup.

oc
```
oc run juice-shop-test --image bkimminich/juice-shop:v9.1.0
oc logs deploymentconfigs/juice-shop-test
```

## :fire: Exception or Error

```txt
> juice-shop@9.1.0 start /juice-shop
> node app

info: All dependencies in ./package.json are satisfied (OK)
info: Detected Node.js version v12.11.1 (OK)
info: Detected OS linux (OK)
info: Detected CPU x64 (OK)
info: Required file index.html is present (OK)
info: Required file styles.css is present (OK)
info: Required file main-es2015.js is present (OK)
info: Required file polyfills-es2015.js is present (OK)
info: Required file runtime-es2015.js is present (OK)
info: Required file vendor-es2015.js is present (OK)
info: Required file main-es5.js is present (OK)
info: Required file polyfills-es5.js is present (OK)
info: Required file runtime-es5.js is present (OK)
info: Required file vendor-es5.js is present (OK)
info: Configuration default validated (OK)
fs.js:130
    throw err;
    ^

Error: EACCES: permission denied, copyfile '/juice-shop/data/static/i18n/ar_SA.json' -> '/juice-shop/i18n/ar_SA.json'
    at Object.copyFileSync (fs.js:1801:3)
    at /juice-shop/lib/startup/restoreOverwrittenFilesWithOriginals.js:14:12
    at Array.forEach (<anonymous>)
    at /juice-shop/lib/startup/restoreOverwrittenFilesWithOriginals.js:13:13
    at f (/juice-shop/node_modules/once/once.js:25:25)
    at Glob.<anonymous> (/juice-shop/node_modules/glob/glob.js:151:7)
    at Glob.emit (events.js:210:5)
    at Glob._finish (/juice-shop/node_modules/glob/glob.js:197:8)
    at done (/juice-shop/node_modules/glob/glob.js:182:14)
    at Glob._processReaddir2 (/juice-shop/node_modules/glob/glob.js:434:12)
    at /juice-shop/node_modules/glob/glob.js:371:17
    at RES (/juice-shop/node_modules/inflight/inflight.js:31:16)
    at f (/juice-shop/node_modules/once/once.js:25:25)
    at Glob._readdirEntries (/juice-shop/node_modules/glob/glob.js:578:10)
    at /juice-shop/node_modules/glob/glob.js:555:12
    at FSReqCallback.oncomplete (fs.js:157:23) {
  errno: -13,
  syscall: 'copyfile',
  code: 'EACCES',
  path: '/juice-shop/data/static/i18n/ar_SA.json',
  dest: '/juice-shop/i18n/ar_SA.json'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! juice-shop@9.1.0 start: `node app`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the juice-shop@9.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

## :deciduous_tree: Your Environment

OpenShift v3.11.0
Juice Shop v9.1.0

--- a/Dockerfile
+++ b/Dockerfile
@@ -25,8 +25,8 @@ RUN addgroup juicer && \
 COPY --from=installer --chown=juicer /juice-shop .
 RUN mkdir logs && \
     chown -R juicer logs && \
-    chgrp -R 0 ftp/ frontend/dist/ logs/ data/ && \
-    chmod -R g=u ftp/ frontend/dist/ logs/ data/
+    chgrp -R 0 ftp/ frontend/dist/ logs/ data/ i18n/ && \
+    chmod -R g=u ftp/ frontend/dist/ logs/ data/ i18n/
 USER juicer
 EXPOSE 3000
 CMD ["npm", "start"]

